api/agent.go

```diff
@@ -508,7 +508,7 @@ func getUserCredentials(usersFilePath string, _ *loadtest.Config) ([]user, error
 				return nil, fmt.Errorf("invalid custom authentication found in %q", email)
 			}
 			authService = split[0]
-			if authService != userentity.AuthenticationTypeOpenID && authService != userentity.AuthenticationTypeMattermost {
+			if authService != userentity.AuthenticationTypeOpenID && authService != userentity.AuthenticationTypeSAML && authService != userentity.AuthenticationTypeMattermost {
 				return nil, fmt.Errorf("invalid custom authentication type %q", authService)
 			}
```

Comment:
Nit: Should this be a switch? It's getting a bit harder to read. Something like 
```suggestion
			switch(authService) {
			case userentity.AuthenticationTypeOpenID:
			case userentity.AuthenticationTypeSAML:
			case userentity.AuthenticationTypeMattermost:
			default :
				return nil, fmt.Errorf("invalid custom authentication type %q", authService)
			}
```